lessons/08-html-site/contact.html

```diff
@@ -0,0 +1,22 @@
+<html>
+    <title>Home|Jaynen Sukhadia</title>
+    
+    <header>
+        <h1>
+            <img src="https://images-na.ssl-images-amazon.com/images/M/MV5BMTAyNTAzMTA4OTJeQTJeQWpwZ15BbWU3MDA4NDI2Njk@._V1_UX172_CR0,0,172,256_AL_.jpg"; width="50px"; height="50px"; alt="My image">
+        </h1>
+        <h2>Jaynen's Portfolio</h2>
+    <nav>
+        <a href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="contact.html">Contact</a>
+    </nav>
+    </header>
+    <hr>
+    <form action="contact-success.html"; method="GET">
+    <p>Name: <input type="text"; name="name1"></p>
+    <p>Email: <input type="email"; name="email"></p>
```

Comment:
You don't need to place a semi-colon `;` after attributes 😄

Reply:
ok.